language-plutus-core/src/Language/PlutusCore/Pretty.hs

```diff
@@ -77,5 +82,23 @@ prettyPlcCondensedErrorClassicString =
     docString . prettyPlcCondensedErrorBy defPrettyConfigPlcClassic
 
 -- | A command suitable for use in GHCi as an interactive printer.
-interactivePrint :: PrettyPlc a => a -> IO ()
-interactivePrint = print . prettyPlcDef
+interactivePrint :: Pretty a => a -> IO ()
+interactivePrint = print . pretty
+
+{- Note [Default pretty instances for PLC]
+While the flexible pretty-printing infrastructure is useful when you want it,
+it's helpful to have an implementation of the default Pretty typeclass that
+does the default thing.
+-}
+instance Pretty (Kind a) where
+    pretty = prettyPlcDef
+instance Pretty (Constant a) where
+    pretty = prettyPlcDef
+instance Pretty (Builtin a) where
+    pretty = prettyPlcDef
```

Comment:
This does not already have a `Pretty` instance?

Reply:
No.

Reply:
Indeed. It's a bit weird, since

```haskell
data Builtin a = BuiltinName a BuiltinName
               | DynBuiltinName a DynamicBuiltinName
```

and both of `BuiltinName` and `DynamicBuiltinName` have their `Pretty` instances.